crates/rolldown_binding/src/options/binding_input_options/mod.rs

```diff
@@ -85,6 +87,7 @@ pub struct BindingInputOptions {
   pub watch: Option<BindingWatchOption>,
   pub keep_names: Option<bool>,
   pub checks: Option<binding_checks_options::BindingChecksOptions>,
+  pub target: Option<BindingTarget>,
```

Comment:
Perhaps we can use `String` to support more inputs?

Reply:
Yes. String is better. We might accept browserlist as the input too.